db/mockdata.py

```diff
@@ -0,0 +1,183 @@
+"""
+This module contains classes required by the compact API.
+"""
+
+import os
+import binascii
+from datetime import datetime, timedelta
+import db
+from sqlalchemy import text
+
+
+# Token expiration time in minutes
+TOKEN_EXPIRATION_TIME = 60
+
+
+class User(object):
+    def __init__(self, row):
+        serial, created, name, auth_token = row
+        self.id = serial
+        self.name = name
+        self.created = created
+        self.api_key = auth_token
```

Comment:
These 5 lines could be compacted into one line by using the construct on line 18.